### ISSUE TITLE
Rename `parseZoneRange` method

### ISSUE THREAD

**Issue by gf3** [rotaready/moment-range#207]
This method is attached to the moment instance and as such should be more descriptive. The name `parseZoneRange` is ambiguous and should reflect what it's actually doing. My suggestion is to follow both our own conventions with `rangeFromInterval` and the [moment convention](https://momentjs.com/docs/#/displaying/as-iso-string/) and use something like, `rangeFromISOString`. Usage:

``` js
// Before
const interval = '2015-01-17T09:50:00+03:00/2015-04-17T08:29:55-04:00';
const range = moment.parseZoneRange(interval);

// After
const interval = '2015-01-17T09:50:00+03:00/2015-04-17T08:29:55-04:00';
const range = moment.rangeFromISOString(interval);
```

**Comment by TristanJM** [comment]
In agreement with this, should be a simple change to be included in 4.0.0

**Comment by gf3** [comment]
Closed by #211
